### PR TITLE
Adding jq and healthcheck scripts to container image.

### PR DESCRIPTION
**Pull request by DuncanDoyle**
Added JQ and the healthcheck scripts to the container image. Removed "yum" from the container image, as it is no longer required.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ ENV RHOAS_CLI_PATH="/usr/local/bin/rhoas"
 ENV OC_CLI_PATH="/usr/local/bin/oc"
 
 # Install required packages
-RUN microdnf install shadow-utils yum
+RUN microdnf install shadow-utils jq
```
Comment (craicoverflow) [inline Dockerfile:21]: Needs a rebase - I have added JQ in a previous PR: https://github.com/redhat-developer/app-services-tools/blob/main/Dockerfile#L27

File: Dockerfile [hunk 2]
```diff
@@ -32,7 +32,9 @@ USER root
 RUN mkdir -p /.config/rhoas && chmod 777 -R /.config/rhoas && echo "{}" > /.config/rhoas/config.json && chmod 777 /.config/rhoas/config.json
 RUN mkdir /.kube && chmod 777 /.kube
 USER rhoas
+WORKDIR /home/rhoas
 
+COPY scripts /home/rhoas/scripts
```
Comment (craicoverflow) [inline Dockerfile:37]: I've added the scripts already, though not to this location: https://github.com/redhat-developer/app-services-tools/blob/main/Dockerfile#L36

**Review by craicoverflow — CHANGES_REQUESTED** [review]
Needs to be rebased.

**Comment by wtrocki** [comment]
Changes were backported.

**Pull request closed without merging**